extensions/amazon-aws/src/step-functions.tsx

```diff
@@ -0,0 +1,59 @@
+import { StateMachineListItem, SFNClient, ListStateMachinesCommand } from "@aws-sdk/client-sfn"; // ES Modules import
+import { Action, ActionPanel, Icon, List } from "@raycast/api";
+import { useCachedPromise } from "@raycast/utils";
+import AWSProfileDropdown from "./components/searchbar/aws-profile-dropdown";
+import { resourceToConsoleLink } from "./util";
+
+export default function stepFunctions() {
```

Comment:
Not important, but it's a convention to uppercase all React Components, so we could do that here, too.

```suggestion
export default function StepFunctions() {
```

Reply:
thanks! It sounds great so I renamed!